08_deploy_bmo.sh

```diff
@@ -2,15 +2,39 @@
 
 eval "$(go env)"
 
+source common.sh
+
 # Get the latest bits for baremetal-operator
 export BMOPATH="$GOPATH/src/github.com/metalkube/baremetal-operator"
 
-oc --as system:admin --config ocp/auth/kubeconfig new-project bmo-project
+# Make a local copy of the baremetal-operator code to make changes
+cp -r $BMOPATH/deploy ocp/.
+sed -i 's/bmo-project/openshift-machine-api/g' ocp/deploy/role_binding.yaml
+
+# Kill the full ironic container on the host since it is performing DHCP and doesn't
+# allow our pod in openshift to take over.
```

Comment:
Looks like a good workaround, we'd probably be better off with a separate container for dnsmaq though so we can just kill that.  I'm working on a PR for separate containers and should have something shortly.  That will also fix the logging problems that Ian tracked in https://github.com/metalkube/metalkube-ironic/issues/13

Reply:
actually this breaks the cleanup.  Ironic loses info about the nodes its deployed.

Probably you can just ignore this whole thing :).